templates/docker-init.j2

```diff
@@ -1,7 +1,7 @@
 # Generated by Ansible for {{ansible_fqdn}}
 description "Docker daemon"
 
-start on filesystem and started lxc-net
```

Comment:
Not sure of the initial need for lxc-net but if anyone needs it (and I assume that originally it was needed) then it would be better if we could at least make it optional using a flag. On a related note, maybe that won't be needed depending on #3 .